common/src/main/java/com/wynntils/models/abilities/type/OphanimOrb.java

```diff
@@ -13,6 +13,15 @@ public OphanimOrb(HealthState healthState) {
         this.healthState = healthState;
     }
 
+    public Integer getHealtState() {
```

Comment:
```suggestion
    public int getHealthState() {
```

Don't forget to update the usage of this if you apply the suggestion

Reply:
Still needs the method name corrected